tests/integration/local/start_api/test_start_api_with_terraform_application.py

```diff
@@ -54,3 +97,38 @@ def test_successful_request(self):
 
         self.assertEqual(response.status_code, 200)
         self.assertEqual(response.json(), {"message": "hello world"})
+
+
+@skipIf(
+    not CI_OVERRIDE,
+    "Skip Terraform test cases unless running in CI",
+)
+@pytest.mark.flaky(reruns=3)
+class TestStartApiTerraformApplicationV1OpenApiAuthorizer(TerraformStartApiIntegrationApplyBase):
```

Comment:
I believe it is better to avoid using V1 on our names, I believe for V2 testing we can just modify the current projects and add some V2 resources for testing.